aws_lambda_powertools/event_handler/middlewares/openapi_validation.py

```diff
@@ -414,6 +597,21 @@ def _handle_missing_field_value(
 
 def _normalize_field_value(value: Any, field_info: FieldInfo) -> Any:
     """Normalize field value, converting lists to single values for non-sequence fields."""
+    # Convert UploadFile to bytes if the expected type is bytes
+    if isinstance(value, UploadFile):
```

Comment:
This function was originally meant to normalize values, not convert them. Maybe we should keep conversions separate to avoid changing its intent.

What do you think?